### DISCUSSION TITLE
Dockers Avalonia Build Tools

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#13537] [Ideas]
On the C.I. Instead of installing the SDKs, Workloads, Nuke and Nuget packages at each build, you could think about creating Dockers containers and use it in C.I. builds.

**Comment by Gillibald** [comment]
Installing SDKs etc. on each build ensures this keeps working. Sometimes these dependencies break or are no longer available. But using some image that already has these dependencies installed could speed up things. Our own Agents could already do that.
